scripts/theme.js

```diff
@@ -95,14 +95,14 @@ $(function(){
 		var $sheet = $('<style>').appendTo('head');
```

Comment:
I'm getting an illegal string was specified error when trying to run this manually.